octobot_evaluators/evaluators/TA_evaluator.py

```diff
@@ -117,13 +117,18 @@ async def evaluators_callback(self,
                 import octobot_trading.api as exchange_api
                 exchange_id = data[constants.EVALUATOR_CHANNEL_DATA_EXCHANGE_ID]
                 symbol_data = self.get_exchange_symbol_data(exchange_name, exchange_id, symbol)
+                time_frame = None
```

Comment:
?

Reply:
Normally it should not be required as KeyError is happening inside the for loop and therefore time_frame is defined. However to be sure, I prefer forcing its init

Reply:
Why not handling it in the error message ?
I find it strange to initialize the variable before the loop